WooCommerce/WooCommerceTests/Model/CouponWooTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+@testable import Networking
+@testable import WooCommerce
+
+final class CouponWooTests: XCTestCase {
+    func test_expiry_status_is_active_if_dateExpires_is_undefined() {
+        // Given
+        let coupon = Coupon.fake().copy(dateExpires: nil)
+
+        // Then
+        XCTAssertEqual(coupon.expiryStatus, .active)
+    }
+
+    func test_expiry_status_is_active_if_dateExpires_is_later_than_current_date() throws {
```

Comment:
nit: this test case doesn't seem to throw an exception?

```suggestion
    func test_expiry_status_is_active_if_dateExpires_is_later_than_current_date() {
```

Reply:
Fixed in fd83ce3ae712df2c3c0f1ac2cfc9c72751bad772 😅